### ISSUE TITLE
Reg\Register an Operation: Descriptions for each Registration Purpose for users

### ISSUE THREAD

**Issue by patriciarussellCAS** [bcgov/cas-registration#2313]
#### Dependencies
- Relates to #2169 
- Blocked by
- Blocking

#### Description:

To help guide external users to select the correct Registration Purpose, we need to add some language on the page that describes each purpose. This will (hopefully) prevent users from selecting the incorrect registration purpose. 

Registration Purposes include: 
- Reporting Operation
- OBPS Regulated Operations
- Opted-in/Opt-in Operation ** [need clarity on whether these need to be 2 separate categories]
- New Entrant Operation
- Electricity Import Operation
- Potential Reporting Operation

[WIREFRAME](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=21269-5894&t=E3hGTXhDevmgcmZk-4)

#### Acceptance Criteria:

Given I am an External Industry users on the Operation Information page in Registration
When I select "**Reporting Operation**" as my Registration Purpose
Then the following language appears below my selection: 

> Note: A Reporting Operation is an industrial operation that emits greater than or equal to 10 000 tCO2e per year and must submit an emission report, but does not produce a regulated product.

Given I am an External Industry users on the Operation Information page in Registration
When I select "**OBPS Regulated Operation**" as my Registration Purpose
Then the following language appears below my selection: 

> "Note: An OBPS Regulated Operation is a type of reporting operation that produces a regulated product and emits greater than or equal to 10 000 tCO2e per year. This option should be selected if you are a registering as a mandatory participant in the B.C. OBPS."

Given I am an External Industry users on the Operation Information page in Registration
When I select "**Opted-in Operation**" as my Registration Purpose
Then the following language appears below my selection: 

> "Note: An Opted-in Operation is an industrial operation than produces a regulated product and emits 
less than 10 000 tCO2e per year. These operations may apply to voluntarily participate in the B.C. OBPS. This option should be selected if you are registering and applying to be an Opted-In Operation." 
****To confirm if any difference is required between those who opted-in in Reg Part 1 and those opting in now. 

Given I am an External Industry users on the Operation Information page in Registration
When I select "**New Entrant**" as my Registration Purpose
Then the following language appears below my selection: 

> Note: This option should be selected if you are registering and applying to be a New Entrant.

Given I am an External Industry users on the Operation Information page in Registration
When I select "**Electricity Import Operation**" as my Registration Purpose
Then the following language appears below my selection: 

> Note: An Electricity Import Operation is an industrial operation that imports electricity from an electricity generating facility located outside British Columbia and is the first point of delivery in British Columbia.

Given I am an External Industry users on the Operation Information page in Registration
When I select "**Potential Reporting Operation**" as my Registration Purpose
Then the following language appears below my selection: 

> Note: A Potential Reporting Operation is an industrial operation that did not submit an emission report for the previous year, but has forecast it's emissions to be greater than or equal to 10 000 tCO2e for the current reporting period. Potential Reporting Operations must register, but do not need to submit an emission report unless they actually emit greater than or equal to 10 000 tCO2e in the current reporting period.

#### Development Checklist:

- [x] Add a condition to display the descriptions below the purpose dropdown based on the selected purpose.
- [x] Add front-end tests
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [ ] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [x] Acceptance criteria are tested by the CI pipeline
- [x] UI meets accessibility requirements
- [x] Configuration changes are documented, documentation and designs are updated
- [x] Passes code peer-review
- [x] Passes QA of Acceptance Criteria with verification in Dev and Test
- [x] Ticket is ready to be merged to main branch
- [x] Can be demoed in Sprint Review
- [x] Bugs or future work cards are identified and created
- [x] Reviewed and approved by Product Owner

**Comment by patriciarussellCAS** [comment]
Reviewed in Dev - works great. 
@andrea-williams can you confirm that all the relevant DOD items were checked? If yes, I will move this to accepted.
